examples/Migration/composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "require": {
-      "googleads/google-ads-php": "^5.0.0",
+      "googleads/google-ads-php": "^14.0.0",
```

Comment:
nit: you could use `>=14.0` to be future proof.

Reply:
Could you do this in your #763?
I don't have access to my usual development environment right now. Thanks.

Reply:
as i already mentionend in my bug report, using v14.0.0 or above will lead to:

`Fatal error: Uncaught Error: Class 'Google\Ads\GoogleAds\Examples\Utils\Helper' not found in /var/www/html/web/google-ads-php/examples/Migration/CampaignManagement/CreateCompleteCampaignAdWordsApiOnly.php on line 118

Error: Class 'Google\Ads\GoogleAds\Examples\Utils\Helper' not found in /var/www/html/web/google-ads-php/examples/Migration/CampaignManagement/CreateCompleteCampaignAdWordsApiOnly.php on line 118`

Since the folder googleads-ads-php/src/Google/Ads/GoogleAds/Example is missing

only the testclass for this Helper is available under

`google-ads-php/tests/Google/Ads/GoogleAds/Examples/Utils/HelperTest.php`

Reply:
as i already mentionend in my bug report, using v14.0.0 or above will lead to:

`Fatal error: Uncaught Error: Class 'Google\Ads\GoogleAds\Examples\Utils\Helper' not found in /var/www/html/web/google-ads-php/examples/Migration/CampaignManagement/CreateCompleteCampaignAdWordsApiOnly.php on line 118`

`Error: Class 'Google\Ads\GoogleAds\Examples\Utils\Helper' not found in /var/www/html/web/google-ads-php/examples/Migration/CampaignManagement/CreateCompleteCampaignAdWordsApiOnly.php on line 118`


Since the folder `googleads-ads-php/src/Google/Ads/GoogleAds/Example` is missing, but its available in v5.0.0, but there on the other hand is the lib v10 missing which is referred in the RunExamples.php file

only the testclass for this Helper is available under

`google-ads-php/tests/Google/Ads/GoogleAds/Examples/Utils/HelperTest.php`

Reply:
I noticed you removed the references to the old example/helper and now it works :dancers: